lib/Doctrine/ORM/Tools/SchemaTool.php

```diff
@@ -947,11 +947,11 @@ public function updateSchema(array $classes, $saveMode = false)
      * Gets the sequence of SQL statements that need to be performed in order
      * to bring the given class mappings in-synch with the relational schema.
      *
-     * @param mixed[] $classes  The classes to consider.
-     * @param bool    $saveMode If TRUE, only generates SQL for a partial update
-     *                           that does not include SQL for dropping assets which are scheduled for deletion.
+     * @param bool $saveMode If TRUE, only generates SQL for a partial update
+     *                       that does not include SQL for dropping assets which are scheduled for deletion.
+     * @psalm-param list<ClassMetadata> $classes  The classes to consider.
```

Comment:
```suggestion
     * @param list<ClassMetadata> $classes  The classes to consider.
```

No need to vendor-prefix this annotation.